products/views.py

```diff
@@ -1,3 +1,62 @@
-from django.shortcuts import render
+import datetime
 
-# Create your views here.
+from django.http  import JsonResponse
+from django.views import View
+
+from dr_martens.checkitem   import CheckItem
+from dr_martens.login_check import login_check
+from products.models        import (Product,
+                                    ProductImage,
+                                    ProductOption)
+
+class DetailView(View):
```

Comment:
```suggestion
class ProductDetailView(View):
```
어떤 데이터의 detail인지 명시해 주면 더 좋을 것 같습니다.

Reply:
리뷰 감사합니다.
ProductDetailView로 변경하면 좀 더 직관적인 클래스명이 되는 것 같습니다.